spec/requests/posts_spec.rb

```diff
@@ -2,6 +2,36 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    it 'Return http success' do
```

Comment:
Has `users_spec.rb` and `posts_spec.rb` in `spec/requests/` directory 🐐Well done!